app/models/Schedule.ts

```diff
@@ -0,0 +1,22 @@
+import type { Service } from './Service';
+
+export interface Schedule {
+  id: number;
+  hours_known: boolean;
+  schedule_days: ScheduleDay[];
+}
+
+export interface ScheduleDay {
+  id: number;
+  day: string;
+  opens_at: number | null;
+  closes_at: number | null;
+}
+
+export interface ScheduleParams extends Omit<Partial<Schedule>, 'schedule_days'> {
+  schedule_days: Partial<ScheduleDay>[];
+}
+
+export const shouldInheritSchedule = (service: Service) => (
```

Comment:
Originall this was done because of circular imports between Service > Org > Service > Org, but the`import type` annotation importing org can solve that, so fixed.